x-pack/plugins/ml/public/application/data_frame_analytics/pages/components/analytics_selector/analytics_id_selector.tsx

```diff
@@ -154,6 +154,7 @@ export function AnalyticsIdSelector({
   async function fetchAnalyticsModels() {
     setIsLoading(true);
     try {
+      // FIXME should if fetch all trained models?
```

Comment:
Yes, this is correct for when we're using the analytics selection flyout from the maps page - we want to allow users to choose jobs or models, which requires us to fetch the models. Though, looks like in `x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/page.tsx` 
the `AnalyticsIdSelector` should have the `jobsOnly` prop set to 'true' so that we don't fetch models for the exploration page - which only pertains to jobs. 
Looks like I missed setting that correctly. Would you be up for adding it? 
So in that file it should be:
```
<AnalyticsIdSelector
      setAnalyticsId={setAnalyticsId}
       setIsIdSelectorFlyoutVisible={setIsIdSelectorFlyoutVisible}
       jobsOnly
  />

Reply:
I mean, it also fetched all NLP models, I believe it doesn't make sense. 
It should only list DFA models.

<img width="1446" alt="image" src="https://github.com/user-attachments/assets/92e0b90e-05ed-41c9-81b2-ec52d82331e5">